website/src/components/docs.js

```diff
@@ -1,12 +1,12 @@
 import cx from 'classnames'
 import { Link, graphql, useStaticQuery } from 'gatsby'
-import { useUpdateAtom } from 'jotai/utils'
+import { useSetAtom } from 'jotai'
```

Comment:
Please don't change this. We can only change this after releasing the new version.